src/dispatch/static/dispatch/src/task/Table.vue

```diff
@@ -58,6 +58,12 @@
                   Unknown
                 </v-chip>
               </template>
+              <template v-slot:item.incident_priority.name="{ item }">
```

Comment:
I think `incident` is missing after `item.`

Reply:
Same as above.